diff --git a/node_modules/react-native/.gradle/8.2/checksums/checksums.lock b/node_modules/react-native/.gradle/8.2/checksums/checksums.lock
new file mode 100644
index 0000000..13ae758
Binary files /dev/null and b/node_modules/react-native/.gradle/8.2/checksums/checksums.lock differ
diff --git a/node_modules/react-native/.gradle/8.2/checksums/sha1-checksums.bin b/node_modules/react-native/.gradle/8.2/checksums/sha1-checksums.bin
new file mode 100644
index 0000000..5efb4dc
Binary files /dev/null and b/node_modules/react-native/.gradle/8.2/checksums/sha1-checksums.bin differ
diff --git a/node_modules/react-native/.gradle/8.2/dependencies-accessors/dependencies-accessors.lock b/node_modules/react-native/.gradle/8.2/dependencies-accessors/dependencies-accessors.lock
new file mode 100644
index 0000000..20c5798
Binary files /dev/null and b/node_modules/react-native/.gradle/8.2/dependencies-accessors/dependencies-accessors.lock differ
diff --git a/node_modules/react-native/.gradle/8.2/dependencies-accessors/gc.properties b/node_modules/react-native/.gradle/8.2/dependencies-accessors/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native/.gradle/8.2/fileChanges/last-build.bin b/node_modules/react-native/.gradle/8.2/fileChanges/last-build.bin
new file mode 100644
index 0000000..f76dd23
Binary files /dev/null and b/node_modules/react-native/.gradle/8.2/fileChanges/last-build.bin differ
diff --git a/node_modules/react-native/.gradle/8.2/fileHashes/fileHashes.lock b/node_modules/react-native/.gradle/8.2/fileHashes/fileHashes.lock
new file mode 100644
index 0000000..e9608cc
Binary files /dev/null and b/node_modules/react-native/.gradle/8.2/fileHashes/fileHashes.lock differ
diff --git a/node_modules/react-native/.gradle/8.2/gc.properties b/node_modules/react-native/.gradle/8.2/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native/.gradle/buildOutputCleanup/buildOutputCleanup.lock b/node_modules/react-native/.gradle/buildOutputCleanup/buildOutputCleanup.lock
new file mode 100644
index 0000000..7a7215e
Binary files /dev/null and b/node_modules/react-native/.gradle/buildOutputCleanup/buildOutputCleanup.lock differ
diff --git a/node_modules/react-native/.gradle/buildOutputCleanup/cache.properties b/node_modules/react-native/.gradle/buildOutputCleanup/cache.properties
new file mode 100644
index 0000000..9ea885e
--- /dev/null
+++ b/node_modules/react-native/.gradle/buildOutputCleanup/cache.properties
@@ -0,0 +1,2 @@
+#Fri Apr 19 16:01:57 ICT 2024
+gradle.version=8.2
diff --git a/node_modules/react-native/.gradle/buildOutputCleanup/outputFiles.bin b/node_modules/react-native/.gradle/buildOutputCleanup/outputFiles.bin
new file mode 100644
index 0000000..90dc857
Binary files /dev/null and b/node_modules/react-native/.gradle/buildOutputCleanup/outputFiles.bin differ
diff --git a/node_modules/react-native/.gradle/vcs-1/gc.properties b/node_modules/react-native/.gradle/vcs-1/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native/index.js b/node_modules/react-native/index.js
index 7149c64..140902f 100644
--- a/node_modules/react-native/index.js
+++ b/node_modules/react-native/index.js
@@ -380,43 +380,18 @@ module.exports = {
   },
   // Deprecated Prop Types
   get ColorPropType(): $FlowFixMe {
-    console.error(
-      'ColorPropType will be removed from React Native, along with all ' +
-        'other PropTypes. We recommend that you migrate away from PropTypes ' +
-        'and switch to a type system like TypeScript. If you need to ' +
-        'continue using ColorPropType, migrate to the ' +
-        "'deprecated-react-native-prop-types' package.",
-    );
     return require('deprecated-react-native-prop-types').ColorPropType;
   },
+
   get EdgeInsetsPropType(): $FlowFixMe {
-    console.error(
-      'EdgeInsetsPropType will be removed from React Native, along with all ' +
-        'other PropTypes. We recommend that you migrate away from PropTypes ' +
-        'and switch to a type system like TypeScript. If you need to ' +
-        'continue using EdgeInsetsPropType, migrate to the ' +
-        "'deprecated-react-native-prop-types' package.",
-    );
     return require('deprecated-react-native-prop-types').EdgeInsetsPropType;
   },
+
   get PointPropType(): $FlowFixMe {
-    console.error(
-      'PointPropType will be removed from React Native, along with all ' +
-        'other PropTypes. We recommend that you migrate away from PropTypes ' +
-        'and switch to a type system like TypeScript. If you need to ' +
-        'continue using PointPropType, migrate to the ' +
-        "'deprecated-react-native-prop-types' package.",
-    );
     return require('deprecated-react-native-prop-types').PointPropType;
   },
+
   get ViewPropTypes(): $FlowFixMe {
-    console.error(
-      'ViewPropTypes will be removed from React Native, along with all ' +
-        'other PropTypes. We recommend that you migrate away from PropTypes ' +
-        'and switch to a type system like TypeScript. If you need to ' +
-        'continue using ViewPropTypes, migrate to the ' +
-        "'deprecated-react-native-prop-types' package.",
-    );
     return require('deprecated-react-native-prop-types').ViewPropTypes;
   },
 };
